neuralmonkey/config/utils.py

```diff
@@ -30,13 +33,13 @@ def dep_func(*args, **kwargs):
 initialize_vocabulary = vocabulary.initialize_vocabulary
 
 
-def adam_optimizer(learning_rate=1e-4):
+def adam_optimizer(learning_rate=1e-4) -> tf.train.AdamOptimizer:
     return tf.train.AdamOptimizer(learning_rate)
 
 
-def adadelta_optimizer(**kwargs):
+def adadelta_optimizer(**kwargs) -> tf.train.AdadeltaOptimizer:
     return tf.train.AdadeltaOptimizer(**kwargs)
 
 
-def variable(initial_value=0, trainable=False, **kwargs):
+def variable(initial_value=0, trainable=False, **kwargs) -> tf.Variable:
```

Comment:
Vskutečnosti tf.Tensor nebo něco, co se na něj dá převést. A to se mi nechce anotovat.